File: Java/Card.java
package Java;

import java.util.Date;

public class Card extends Payment {
    String number;
    String cvv;
    Date date;
    
    public Card(Integer id, String number, String cvv, Date date){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}

File: PHP/Card.php
<?php
require_once('Payment.php');

class Card extends Payment {
    public $number;
    public $cvv;
    public $date;

    function __construct($id, $number, $cvv, $date) {
        parent::__construct($id);
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }

    public function PrintDataCard() {
        echo "number: $this->number, cvv: {$this->cvv}, date: $this->date";
    }
}